### ISSUE TITLE
Disconnecting with an error: Error: ERROR_QUEUE_NOT_FOUND

### ISSUE THREAD

**Issue by kirbo** [jgeboski/bitlbee-facebook#15]
Didn't know which issue this would be related to, so created a new one. Close this issue if needed.

I updated to "development" branch, since I noticed there were some fixes for #9, and now my bitlbee-facebook is pretty much as unstable as it can be. Some debug:
```
[09/03 22:28:35] ùíù Netsplit domain.name <-> email-account quits: friend1, friend2, friend3, friend4, friend5, ... (+11 more, use /NETSPLIT to show all of them)
[09/03 22:28:35] <@root> facebook - Reconnecting in 5 seconds..
[09/03 22:28:40] <@root> facebook - Logging in: Connecting
[09/03 22:28:40] <@root> facebook - Logging in: Fetching contacts
[09/03 22:28:43] <@root> facebook - Logging in: Establishing connection
[09/03 22:28:45] <@root> facebook - Logging in: Logged in
[09/03 22:28:45] ùíù Netsplit over, joins: friend1, friend2, friend3, friend4, friend5, ... (+11 more)
[09/03 22:28:45] ùíù friend [id@facebook] has joined &bitlbee
[09/03 22:28:48] ùíù Netsplit over, joins: friend1, friend2, friend3
[09/03 22:29:42] <@root> facebook - Error: ERROR_QUEUE_NOT_FOUND
[09/03 22:29:42] <@root> facebook - Signing off..
[09/03 22:29:42] ùíù Netsplit domain.name <-> email-account quits: friend1, friend2, friend3, friend4, friend5, ... (+11 more, use /NETSPLIT to show all of them)
[09/03 22:29:42] <@root> facebook - Reconnecting in 5 seconds..
[09/03 22:29:47] <@root> facebook - Logging in: Connecting
[09/03 22:29:47] <@root> facebook - Logging in: Fetching contacts
[09/03 22:29:51] <@root> facebook - Logging in: Establishing connection
[09/03 22:29:53] <@root> facebook - Logging in: Logged in
[09/03 22:29:53] ùíù Netsplit over, joins: friend1, friend2, friend3, friend4, friend5, ... (+11 more)
[09/03 22:29:53] ùíù friend [id@facebook] has joined &bitlbee
[09/03 22:29:59] ùíù Netsplit over, joins: friend1, friend2, friend3, friend4
[09/03 22:30:12] <@root> facebook - Error: ERROR_QUEUE_NOT_FOUND
[09/03 22:30:12] <@root> facebook - Signing off..
[09/03 22:30:12] ùíù Netsplit domain.name <-> email-account quits: friend1, friend2, friend3, friend4, friend5, ... (+11 more, use /NETSPLIT to show all of them)
[09/03 22:30:12] <@root> facebook - Reconnecting in 5 seconds..
[09/03 22:30:17] <@root> facebook - Logging in: Connecting
[09/03 22:30:17] <@root> facebook - Logging in: Fetching contacts
[09/03 22:30:21] <@root> facebook - Logging in: Establishing connection
[09/03 22:30:22] <@root> facebook - Logging in: Logged in
[09/03 22:30:23] ùíù Netsplit over, joins: friend1, friend2
[09/03 22:30:23] ùíù friend [id@facebook] has joined &bitlbee
```

**Comment by kirbo** [comment]
Here is where it started:
```
```

**Comment by jgeboski** [comment]
Did you remove all of your chats on the Facebook website?

**Comment by kirbo** [comment]
No I didn't do anything in facebook website near this time, I wasn't even awake.
Is there any kind of data/log/anything I could provide you which would help tracing this the cause of this issue?

**Comment by jgeboski** [comment]
Try readding the account.  This will wipe-out all the session specific information.

**Comment by kirbo** [comment]
I did try readding my account before, with no luck (atleast didn't work that long), but now it seems like that did the trick.

Thanks :)

You may close this issue if you feel like closing it, I will leave this open (at least for now) since in my opinion there is no information what caused this/how this could be prevented.

**Comment by dequis** [comment]
@seirl reported the same issue earlier today

**Comment by Zopieux** [comment]
I too can report the same issue today. Flooded my bitlbee buffer with 5 sec connection retries…

**Comment by seirl** [comment]
Cycling the account does not work at all for me.

**Comment by Zopieux** [comment]
@seirl There was a several hour delay between the `off` and the `on`. Might be the real fix.

**Comment by dequis** [comment]
FWIW deleting and readding the account probably worked because bitlbee-facebook creates a queue the first time it logs in (when `stoken`, which is saved as a setting, is null).

My PR, #16, changes that to also create a queue when getting this error.

**Comment by jgeboski** [comment]
I just merged a patch from @dx into `development` which should resolve this issue.

**Comment by dequis** [comment]
Pinged the wrong dx there :D

**Comment by jgeboski** [comment]
This has been resolved as of 8499f80.
